src/main/java/com/revature/model/InterviewInterviewer.java

```diff
@@ -0,0 +1,83 @@
+package com.revature.model;
+import java.util.Objects;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "interview_interviewer")
+public class InterviewInterviewer {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name="interview_interviewer_id")
+	private int id;
```

Comment:
Add a blank line between each member variable for readability.

Reply:
And between methods as well.

Reply:
Changes are made as per suggestion